README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
```

Comment:
Unfortunately I have to agree with these comments. "Use the source Luke" in the 90s was what people said of software that had neither documentation nor very good code comments. When I first saw it, it made me look at other options, including impress.js accessories like hovercraft. Only later did I actually look at the HTML source and realized it reads as a tutorial.

Ideally of course, there should be both documentation, and the source code comments. But already in current state, this wording could be made in a better way.